### ISSUE TITLE
Can't handle multiple llined MetaDescription

### ISSUE THREAD

**Issue by normann** [chrometoasters/silverstripe-metadescription-fallback#5]
when the page's MetaDescription  has multiple lined text, it creates 2 meta data with name="description". e.g.
if my MetaDescription (textarea field) is:
```
## How [lang=mi]tatou[/lang] _work_, or [lang=zh-Hans]中文[/lang] `example` page title
Bullet list:
  * How1 [lang=mi]tatou[/lang] _work_, or [lang=zh-Hans]中文[/lang] `example` page title
```

This will results 2 meta data with the same name attribute "description"  as:
```
<meta name="description" content="## How [lang=mi]tatou[/lang] _work_, or [lang=zh-Hans]中文[/lang] `example` page title
  Bullet list:
   * How1 [lang=mi]tatou[/lang] _work_, or [lang=zh-Hans]中文[/lang] `example` page title" />
```
and
```
<meta name="description" content="## How [lang=mi]tatou[/lang] _work_, or [lang=zh-Hans]中文[/lang] `example` page title Bullet list: * How1 [lang=mi]tatou[/lang] _work_, or [lang=zh-Hans]中文[/lang] `example` page title" />
```

**Comment by normann** [comment]
In the example above The first meta data is created by default MetaData function in the core, the second is appended by the fallback function in this module, so there are 2 issues in the above out put,
issue one: the second one should `replace` the first one, rather than `append`
issue two: the string in content attribute should use the right format from DBText::ATT()

**Comment by normann** [comment]
Fixed by PR #6 (approved and  merged)
